jwst/source_catalog/source_catalog.py

```diff
@@ -917,8 +916,8 @@ def format_columns(catalog):
                 catalog[colname].info.format = '.4f'
             if 'flux' in colname:
                 catalog[colname].info.format = '.6e'
-            if ('abmag' in colname or 'vegamag' in colname
-                    or colname in ('nn_dist', 'sharpness', 'roundness')):
+            if ('abmag' in colname or 'vegamag' in colname or
+                    colname in ('nn_dist', 'sharpness', 'roundness')):
```

Comment:
Same here.

Reply:
At https://peps.python.org/pep-0008/ in the section on conditionals it appears the or should go at the end of the preceding line, instead of the beginning of the next line.